modules/configuring-the-codeflare-operator.adoc

```diff
@@ -0,0 +1,102 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-codeflare-operator_{context}"]
+= Configuring the CodeFlare Operator
+
+[role='_abstract']
+If you want to change the default configuration of the CodeFlare Operator for distributed workloads in {productname-short}, you can edit the associated config map.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+
+.Procedure
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, click *Workloads* -> *ConfigMaps*.
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, click *Workloads* -> *ConfigMaps*.
+endif::[]
+
+ifdef::self-managed,cloud-service[]
+. From the *Project* list, select *redhat-ods-applications*.
+endif::[]
+ifdef::upstream[]
+. From the *Project* list, select *odh*.
+endif::[]
+
+. Search for the *codeflare-operator-config* config map, and click the config map name to open the *ConfigMap details* page.
+
+. Click the *YAML* tab to show the config map specifications.
+. In the `*data*` > `*config.yaml*` > `*kuberay*` section, you can edit the following entries:
```

Comment:
I have not seen this format for a YAML section path before 
- maybe because there is a full-stop in the config.yaml section title?

Reply:
I think this method is clearer for the user. I've added it to the team agenda to discuss.